CONTRIBUTING.md

```diff
@@ -1,3 +1,220 @@
+
+2. Create a new issue and **read the issue template in its entirety and fill out all applicable sections**. If you aren't sure how to create an issue, you can read the GitHub documentation on [creating an issue from a repository](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-repository).
+   * The title of the issue should follow a format similar to `file, exercise/lesson, or folder: brief description of issue`. This makes it easier for anyone to quickly see what an issue is for, reducing the possibility of a duplicate issue from being made.
+   * If you would like to be assigned the issue you are creating, complete the applicable checkbox in the issue template. Note that this does not guarantee that you will be assigned the issue, but rather it lets maintainers know that you are interested.
+
+### Setting Up Your Local Clone
+
+Before you begin working on anything, make sure you follow these steps in order to set up a clone on your local machine:
+
+1. Fork this repo to your own GitHub account. If you don't know how to do so, follow the GitHub documentation on how to [fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+
+2. Clone this repo to your local machine with one of the commands below. Be sure the `<your username>` text is replaced with your actual GitHub username, and the `<repo name>` with the actual repo name. You can also read the GitHub documentaiton on [cloning a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
+
+    ```bash
+    # If you have SSH set up with Git:
+    git clone git@github.com:<your username>/<repo name>.git
+    # Otherwise for HTTPS:
+    git clone https://github.com/<your username>/<repo name>.git
+    
+    # An example:
+    git clone git@github.com:Odinson/css-exercises.git
+    ```
+
+3. `cd` into the directory of your local clone, then set the upstream remote so you can keep your local clone synced with TOP's original repo. The `<repo name>` below should be the same as the one you used when creating your local clone in the previous step.
+
+    ```bash
+    # If you have SSH set up with Git:
+    git remote add upstream git@github.com:TheOdinProject/<repo name>.git
+    # Otherwise for HTTPS:
+    git remote add upstream https://github.com/TheOdinProject/<repo name>.git
+    
+    # An example:
+    git remote add upstream https://github.com/TheOdinProject/css-exercises.git
+    ```
+
+### Working on an Issue
+
+Once you have the repo forked and cloned, and the upstream remote has been set, you can begin working on your issue:
+
+1. Create a new branch, replacing the `<your branch name>` with an actual branch name:
+
+    ```bash
+    git checkout -b <your branch name>
+    
+    # An example:
+    git checkout -b flex_exercises
```

Comment:
I opted to remove those examples just because it seemed like most people named their branches whatever anyway (including maintainers). That is to say, I don't really think we *need* to have people creating branches with as specific names as `fix/something_here` or `feature/something_here`. As long as the branch very briefly describes the overall change being made or what it's related to, I think it should be fine.

Which that I could actually edit into the instructions here: telling users their branch name can really be whatever but it should be succinct and describe the change in some way with a few examples.

Reply:
I was just curious is all. Not enforcing branch names make sense. Thanks for elaborating!

Reply:
Added a little bit of verbiage and a couple other examples. Let me know if you think anything else might be helpful to include!